console/src/components/Services/Data/Common/Components/TableRow.tsx

```diff
@@ -24,6 +28,12 @@ const getColumnInfo = (col: TableColumn, prevValue?: unknown) => {
 
   let columnValueType;
   switch (true) {
+    case clone &&
+      clone[col.column_name] &&
+      !isDisabled &&
+      !['uuid', 'timestamptz'].includes(col.udt_name):
+      columnValueType = 'value';
+      break;
     case !isEditing && (isIdentity || hasDefault || isGenerated):
```

Comment:
We can simplify it.
Instead of adding another `case` we can handle it along with the first case as:
```
case !isEditing && !clone && (isIdentity || hasDefault || isGenerated):
```
Notice added `!clone` part.